darshan-util/pydarshan/darshan/experimental/plots/data_access_by_filesystem.py

```diff
@@ -0,0 +1,405 @@
+"""
+Draft utility code for the `data access by category` section
+of Phil's hand drawing of future report layout.
+"""
+
+import pathlib
+from typing import List, Dict, Optional, Any, Callable
+
+import numpy as np
+import pandas as pd
+import darshan
+
+
+def process_byte_counts(df_reads, df_writes):
+    """
+    Given separate dataframes for read/write IO activity,
+    returns separate pandas Series objects with the counts
+    of bytes read or written per filesystem root.
+
+    Paramaters
+    ----------
+
+    df_reads: a ``pd.DataFrame`` where each row presents
+    at least 1 byte of read-related IO activity, and where
+    additional columns for ``filesystem_root`` and 
+    ``filepath`` are present
+
+    df_reads: a ``pd.DataFrame`` where each row presents
```

Comment:
I just noticed this -- `df_reads` is repeated for both entries

Reply:
Thanks, fixed and marking as resolved.